projects/vdk-heartbeat/.gitlab-ci.yml

```diff
@@ -28,6 +28,28 @@ vdk-heartbeat-test:
       junit: tests.xml
 
 
+vdk-heartbeat-release-acceptance-test:
+  stage: pre_release
+  before_script:
+    - cd projects/vdk-heartbeat
+  script:
+    - set -x
+    - python setup.py sdist --formats=gztar
+    - pip install vdk-heartbeat --extra-index-url $PIP_EXTRA_INDEX_URL
```

Comment:
Why are we installing heartbeat from a private repo?

Reply:
what private repo ?